### PR TITLE
feat: animate hero section

### LINKED ISSUE #204: FeedAHelp: Animate the Hero with 5 emotional slogans with 5 different Images
Image

### PR DESCRIPTION
**Pull request by TanvirAlam**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #204

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/feedahelp/src/components/ui/features/hero/data.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+export const items = [
+  {
+    imgSrc: "https://source.unsplash.com/ezSFnAFi9hY/500x500",
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/hero/data.ts]: Take few images from here:
https://drive.google.com/drive/folders/14xwGSmw5BJOPXyKcPwydIEZ2bulU6Z0O?usp=drive_link, add them in `public` and take it from there
we will later have CMS ticket for this

File: apps/feedahelp/src/components/ui/features/hero/Hero.tsx [hunk 2]
```diff
@@ -1,20 +1,170 @@
...
+  }, [activeItem]);
+
+  useEffect(() => {
+    positionDisplayImg();
+    setActiveItem(0);
+    setAutoScroll();
+  }, []);
+
+  const handleClick = (index: number) => {
+//    listItemsRef.current[activeItem].classList.remove("active");
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/hero/Hero.tsx]: delete this comment if you dont need

File: apps/feedahelp/src/components/ui/features/hero/Hero.styled.tsx [hunk 3]
```diff
@@ -1,26 +1,155 @@
...
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
   }
 `;
 
+const activeItemBorder = styled.div`
+  pointer-events: none;
+  position: absolute;
+  border: 5px solid #ed6c37;
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/hero/Hero.styled.tsx]: few `px`, change them to `rem`

**Review by TanvirAlam — COMMENTED** [review]
Do the changes

File: apps/feedahelp/src/components/ui/features/hero/data.ts (hunk 1 above)
Comment (Rakibs511) [reply apps/feedahelp/src/components/ui/features/hero/data.ts]: done

File: apps/feedahelp/src/components/ui/features/hero/Hero.tsx (hunk 2 above)
Comment (Rakibs511) [reply apps/feedahelp/src/components/ui/features/hero/Hero.tsx]: yeah, deleted. this not needed now

File: apps/feedahelp/src/components/ui/features/hero/Hero.styled.tsx (hunk 3 above)
Comment (Rakibs511) [reply apps/feedahelp/src/components/ui/features/hero/Hero.styled.tsx]: done

**Review by TanvirAlam — COMMENTED** [review]
LGTM @urmirahman  Approve this

**Pull request merged**